cupy/linalg/norms.py

```diff
@@ -6,6 +6,30 @@
 from cupy.linalg import util
 
 
+def _multi_svd_norm(x, row_axis, col_axis, op):
+    """Compute a function of the singular values of the 2-D matrices in `x`.
+    This is a private utility function used by `cumpy.linalg.norm()`.
+    Parameters
+    ----------
+    x : cupy.ndarray
+    row_axis, col_axis : int
+        The axes of `x` that hold the 2-D matrices.
+    op : callable
+        This should be either `cupy.amin` or `cumpy.amax` or `cumpy.sum`.
+    Returns
+    -------
+    result : float or cupy.ndarray
+        If `x` is 2-D, the return values is a float.
+        Otherwise, it is an array with ``x.ndim - 2`` dimensions.
+        The return values are either the minimum or maximum or sum of the
+        singular values of the matrices, depending on whether `op`
+        is `cupy.amin` or `cupy.amax` or `cupy.sum`.
+    """
```

Comment:
I'd suggest removing the entire docstring for a few reasons:
1. This is only a private function and not exposed to end users
2. This helper function is simple enough (only two lines)
3. The docstring format does not match the CuPy (or Google) style

For the docstring style, you can take a look at `norm()`'s docstring right below for how to properly format it.

Reply:
I have removed that docstring.
And I will take care of that docstring convention next time onwards : )